checkout_session.go

```diff
@@ -429,7 +444,8 @@ type CheckoutSessionTotalDetailsBreakdownDiscount struct {
 // CheckoutSessionTotalDetailsBreakdownTax represent the details of tax rate applied to a session.
 type CheckoutSessionTotalDetailsBreakdownTax struct {
 	Amount  int64    `json:"amount"`
-	TaxRate *TaxRate `json:"tax_rate"`
+	Rate    *TaxRate `json:"rate"`
```

Comment:
Why do we have `Rate` appear here?

Reply:
Ah great question, need to call this out in the changelog and should add a comment. This is a bugfix.

The json field is `tax_rate`, not `rate`, so `TaxRate` will never be populated and will always be nil.

https://github.com/stripe/stripe-node/blob/master/types/2020-08-27/Checkout/Sessions.d.ts#L724